### PR TITLE
ELY-2630, EL-2365 , ELY-2635

### PR DESCRIPTION
**Pull request by nidhiazad**
https://issues.redhat.com/browse/ELY-2630,   https://issues.redhat.com/browse/ELY-2635, https://issues.redhat.com/browse/ELY-2635

### CODE REVIEW

**Comment by skyllarr** [comment]
@nidhiazad There are 2 commits with the same title, please make sure there is only 1 commit for 1 issue. Let us know if you need help with that. Thank you so much!

**Comment by PrarthonaPaul** [comment]
Hi @nidhiazad it seems that some tests are failing currently. Would you be able to run `mvn clean install` with `-X` and `-e` tags on your local branch to see what is going wrong? If you need help with debugging, please feel free to let us know.

**Comment by nidhiazad** [comment]
I will check the issue and also run mvn clean install with -X and -e tags on my local branch

File: auth/client/src/main/java/org/wildfly/security/auth/client/AuthenticationConfiguration.java [hunk 1]
```diff
@@ -1147,10 +1147,7 @@ public AuthenticationConfiguration useSaslProtocol(String saslProtocol) {
     }
 
     public AuthenticationConfiguration useWebServices(Map<String, ?> webservicesProperties) {
-        final HashMap<String, Object> newMap = new HashMap<>();
-        newMap.putAll(webservicesProperties);
-        newMap.values().removeIf(Objects::isNull);
-        return new AuthenticationConfiguration(this, SET_WEBSERVICES_PROPS, optimizeMap(newMap));
+        return useWebServicesProperties(webServicesProperties);
```
Comment (skyllarr) [inline auth/client/src/main/java/org/wildfly/security/auth/client/AuthenticationConfiguration.java:1150]: @nidhiazad I am very sorry, this issue should be closed as "won't do", this is not thread safe operation. Please close the jira issue ELY-2630 as won't do, thank you!

Comment (nidhiazad) [reply auth/client/src/main/java/org/wildfly/security/auth/client/AuthenticationConfiguration.java:1150]: okay. Thanks for the update.

Comment (skyllarr) [reply auth/client/src/main/java/org/wildfly/security/auth/client/AuthenticationConfiguration.java:1150]: @nidhiazad Thank you for closing the jira issue. You can remove this commit from this PR. Other 2 issues seem okay to me, only you need to rename the commit that fixes the link to jira issues. Let us know if you need any help, thank you!

**Comment by skyllarr** [comment]
@nidhiazad  ELY-2635 was already merged, and ELY-2630 was closed. This PR contains changing of link from https://issues.jboss.org/browse/ELY to https://issues.redhat.com/browse/ELY , which is issue number ELY-2412.

Please update this PR to contain only the ELY-2412 change in a single commit. Thank you!

**Comment by PrarthonaPaul** [comment]
There haven't been any updates on this one in a few weeks so going to close it. Feel free to re-open. Thanks.
Changes made for [ELY-2635] has been moved to another PR: https://github.com/wildfly-security/wildfly-elytron/pull/2012/

**Pull request closed without merging**